docs/_sources/real_time_messaging.rst.txt

```diff
@@ -27,7 +27,7 @@ Connecting to the Real Time Messaging API
           print sc.rtm_read()
           time.sleep(1)
   else:
-      print "Connection Failed, invalid token?"
+      print "Connection Failed"
```

Comment:
ditto for the `docs/.doctrees` directory. seems like something that should be ignored too.